apps/s_cb.c

```diff
@@ -752,6 +752,22 @@ int verify_cookie_callback(SSL *ssl, const unsigned char *cookie,
 
     return 0;
 }
+
+int generate_stateless_cookie_callback(SSL *ssl, unsigned char *cookie,
+                                       size_t *cookie_len)
+{
+    unsigned int temp;
+    int res = generate_cookie_callback(ssl, cookie, &temp);
+    *cookie_len = temp;
+    return res;
+}
+
+int verify_stateless_cookie_callback(SSL *ssl, const unsigned char *cookie,
+                                     size_t cookie_len)
+{
+    return verify_cookie_callback(ssl, cookie, cookie_len);
```

Comment:
Why do we not need to check that ```cookie_len <= UINT_MAX``` to match the ```verify_cookie_callback()``` prototype?

Reply:
A TLS1.3 cookie is at most 2^16-1 bytes. Is that not a sufficiently strong bound?

Reply:
Probably, assuming the state machine checks that before calling into us (which I'm sure it does because it uses the PACKET functions).